GitUI/CommandsDialogs/RevisionDiffControl.cs

```diff
@@ -902,5 +927,357 @@ public void SwitchFocus(bool alreadyContainedFocus)
                 DiffFiles.Focus();
             }
         }
+
+        /// <summary>
+        /// Hotkey handler
+        /// </summary>
+        /// <returns>true if executed</returns>
+        private bool StageSelectedFile()
+        {
+            if (DiffFiles.Revision.ObjectId == ObjectId.IndexId)
+            {
+                return false;
+            }
+
+            if (DiffText.ContainsFocus && DiffText.SupportLinePatching)
+            {
+                StageOrCherryPickSelectedLines();
+                return true;
+            }
+            else if (DiffFiles.Focused && DiffFiles.Revision.ObjectId == ObjectId.WorkTreeId)
```

Comment:
nit: `else` can be omitted
```suggestion
            
            if (DiffFiles.Focused && DiffFiles.Revision.ObjectId == ObjectId.WorkTreeId)
```

Reply:
ok for all